Chaincase.Common/Global.cs

```diff
@@ -381,7 +442,7 @@ public async Task OnResuming()
                     Nodes.Connect();
                     Logger.LogInfo($"{nameof(Global)}.OnResuming():Start connecting to nodes...");
 
-                    var requestInterval = (Network == Network.RegTest) ? TimeSpan.FromSeconds(5) : TimeSpan.FromSeconds(30);
+                    var requestInterval = (Network == Network.RegTest) ? TimeSpan.FromSeconds(5) : TimeSpan.FromSeconds(3);
```

Comment:
as above "Problem: connection confirmation timeout must be reduced to fit CoinJoin within 30s window, however this background confirmation timeout also dictates how often it expects the client Alice to check in. Less often and we get dropped registration. dropping registration status during InputRegistration is more of an issue with the UI than a practical one since coinjoin is triggered by a push notification. Polling this often may overwhelm the coordinator" ~ we don't want to ALWAYS poll in 3s intervals

Reply:
https://github.com/chaincase-app/Chaincase/issues/202